packages/frontend/contexts/GlobalStaticContext/getStaticGoogleScholarData.ts

```diff
@@ -0,0 +1,142 @@
+import { environment } from "@wdp/lib/app";
+import { fetchQuery, graphql, readInlineData } from "relay-runtime";
+import { routeQueryArrayToString } from "@wdp/lib/routes";
+import { GetServerSidePropsContext } from "next";
+import { getStaticGoogleScholarDataQuery } from "@/relay/getStaticGoogleScholarDataQuery.graphql";
+import { getStaticGoogleScholarDataFragment$key } from "@/relay/getStaticGoogleScholarDataFragment.graphql";
+
+export default async function getStaticGoogleScholarData(
+  context: GetServerSidePropsContext
+) {
+  const { query: urlQuery } = context;
+
+  const slug = routeQueryArrayToString(urlQuery?.slug);
+
+  if (!slug) return;
+
+  const env = environment();
+
+  const data = await fetchQuery<getStaticGoogleScholarDataQuery>(env, query, {
+    slug,
+  }).toPromise();
+
+  if (data) {
+    return readInlineData<getStaticGoogleScholarDataFragment$key>(
+      fragment,
+      data.item || null
+    );
+  }
+}
+
+const query = graphql`
+  query getStaticGoogleScholarDataQuery($slug: Slug!) {
+    item(slug: $slug) {
+      ...getStaticGoogleScholarDataFragment
+    }
+  }
+`;
+
+const fragment = graphql`
```

Comment:
as far as rewriting the fragment? i don't think so. whether anything could be changed on the API side, I don't know, it would require some investigation.

ostensibly, we could cache google scholar tags on the server and handle the generation during lifecycle methods for any given entity type / maybe schema, then you could just pull down something like:

```graphql
item {
  scholarTags {
    key
    value
  }
}
```

i'd need the tags itemized and like, however you're using this data to template them.